docs/scripts_md/batch_run_pipeline_qc_face_script.md

```diff
@@ -0,0 +1,34 @@
+# NAME
```

Comment:
I think the `.md` file is outdated and should be generated again.

Reply:
done